src/tracking.F90

```diff
@@ -137,6 +142,9 @@ subroutine transport(p)
              distance * material_xs % nu_fission
       end if
 
+      ! Score flux derivative accumulators for differential tallies.
+      if (active_tallies % size() > 0) call score_track_derivative(p, distance)
```

Comment:
Nope.  Diff tallies with collision estimators still need to account for how the flux changes as the particle tracks through a medium.